src/widgets/JSONForm/JSONForm.js

```diff
@@ -149,10 +151,13 @@ export const JSONForm = React.forwardRef(
     return (
       <JSONFormContext.Provider value={options}>
         <Form
-          validate={(formData, errorHandlers) => {
+          liveValidate
+          onChange={onChange}
```

Comment:
So this is live validating now and keeping the `formData` as state in the `NameNote` reducer and constantly feeds it back into the component. I thought to update the `formData` in state mutably so that it did not cause re-renders, but it wasn't too much of a performance hit so figured not doing weird stuff for performance wasn't needed.